lib/rucio/api/request.py

```diff
@@ -135,3 +135,22 @@ def get_request_by_did(scope, name, rse, issuer):
     req = request.get_request_by_did(scope, name, rse_id)
 
     return api_update_return_dict(req)
+
+
+def list_requests(src_rse, dst_rse, state, issuer):
```

Comment:
can you make state --> states ? so we can ask for a list of request states?

Reply:
ah yes, of course